src/index.js

```diff
@@ -0,0 +1,362 @@
+import data from './js/Data';
+import Key from './js/Key';
+
+const unpressShift = () => {
+  document.querySelectorAll('.symbol_unshift').forEach((symbol) => {
+    symbol.classList.remove('hide');
+  });
+  document.querySelectorAll('.symbol_shift').forEach((symbol) => {
+    symbol.classList.add('hide');
+  });
+};
+const pressShift = () => {
+  document.querySelectorAll('.symbol_unshift').forEach((symbol) => {
+    symbol.classList.add('hide');
+  });
+  document.querySelectorAll('.symbol_shift').forEach((symbol) => {
+    symbol.classList.remove('hide');
+  });
+};
+
+// render wrapper
+const renderWrapper = () => {
+  const wrapper = document.createElement('div');
+  wrapper.className = 'wrapper';
+  document.body.append(wrapper);
+  return wrapper;
+};
+// textarea rendering
+const renderTextarea = (wrapper) => {
+  const textarea = document.createElement('textarea');
+  textarea.id = 'textarea';
+  textarea.setAttribute('rows', '15');
+  wrapper.append(textarea);
+};
+// Keyboard rendering
+const renderRows = (dataKeyboard) => {
+  if (!localStorage.getItem('language')) {
+    localStorage.setItem('language', 'eng');
+  }
+  const dataKeys = dataKeyboard[localStorage.getItem('language')];
+  const rows = [];
+  dataKeys.forEach((dataRow) => {
+    const row = document.createElement('section');
+    row.className = 'row';
+
+    dataRow.forEach((dataKey) => {
+      const key = new Key(dataKey);
+      const button = key.generateKey();
+      row.append(button);
+    });
+    rows.push(row);
+  });
+  return rows;
+};
+const renderKeyBoard = () => {
+  const keyboard = document.createElement('section');
+  keyboard.className = 'keyboard';
+
+  const rows = renderRows(data);
+  rows.forEach((row) => {
+    keyboard.append(row);
+  });
+  return keyboard;
+};
+//---------------------------------------------
+// change Language
+const changeLanguage = (e) => {
+  e.preventDefault();
+  if (localStorage.getItem('language') === 'eng') {
+    localStorage.setItem('language', 'ru');
+  } else {
+    localStorage.setItem('language', 'eng');
+  }
+  document.querySelector('.keyboard').remove();
+  window.onload();
+};
+// mouse events..
+const mouseKeyUp = (e) => {
+  e.preventDefault();
+  const target = e.currentTarget;
+  let key = '';
+  if (!target) return;
```

Comment:
No matter how many lines are in if statements, always use {}.
![image](https://user-images.githubusercontent.com/31600165/79686190-2ef02380-8247-11ea-94f2-42061a3cf96f.png)